utility/ssd_writeback_cache.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+import os
+import random
+import subprocess
+import sys
+import time
+
+from docopt import docopt
+
+log = logging.getLogger(__name__)
+
+logging.basicConfig(
+    handlers=[logging.StreamHandler(sys.stdout)],
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s",
+)
+
+doc = """
+
+This script helps to validate the write back cache flushing of data
+
+usage:
+SSD_writeback_cache.py  --pool POOL --image IMAGE [--size=<size>] [--iteration=<iteration>]
+Example:
+    python3 test_writeback.py --pool test --image test123 --size=10 --iteration=10
+Options:
+  -h --help                 Show this screen
+  --pool <name>             Name of the pool
+  --image <name>            Image name
+  --size=<size>             Size of the image default is 1G     [default: 1]
+  --iteration=<iteration>   No of iterations for Ios to run     [default: 10000]
+"""
+
+
+def ssd_write_cache(args):
```

Comment:
I think we might not need parameter `args`.

Reply:
removed and tested. its working fine